core/modules/core/src/com/haulmont/yarg/formatters/impl/HtmlFormatter.java

```diff
@@ -118,24 +118,6 @@ protected void renderPdfDocument(String htmlContent, OutputStream outputStream)
         }
     }
 
-    /**
-     * @deprecated
-     * @see #loadFonts(HtmlToPdfConverter)
-     */
-    @Deprecated
-    protected void loadFonts(ITextRenderer renderer) {
-        loadFonts(new ITextPdfConverter(renderer));
-    }
-
-    /**
-     * @deprecated
-     * @see #loadFontsFromDirectory(HtmlToPdfConverter, java.io.File)
-     */
-    @Deprecated
-    protected void loadFontsFromDirectory(ITextRenderer renderer, File fontsDir) {
-        loadFontsFromDirectory(new ITextPdfConverter(renderer), fontsDir);
-    }
-
     protected void loadFonts(HtmlToPdfConverter converter) {
```

Comment:
Please, do not remove deprecated methods. Methods are used for backward compatibility.